src/views/ToDo.vue

```diff
@@ -47,18 +47,14 @@ export default {
     emailContent: function() {
       const subject = 'Earthlife';
       let body = `I pledge to:
-
-${this.todos.map(todo => '- ' + todo.text).join('\r\n')}
-
-
-http://earthfestsingapore.com`;
-
+      ${this.todos.map(todo => '- ' + todo.text).join('\r\n')}http://earthfestsingapore.com`;
```

Comment:
This block cannot be reformatted/removed/changed because the newlines are needed for the email content